recipes/charls/all/conanfile.py

```diff
@@ -110,9 +110,8 @@ def package(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "charls")
         self.cpp_info.set_property("cmake_target_name", "charls")
-        self.cpp_info.builddirs.append(self._module_subfolder)
-        self.cpp_info.set_property("cmake_build_modules", [self._module_file_rel_path])
```

Comment:
this PR does not change target names, but removes `cmake_build_modules` property. Is it intended ? is the title inaccurate?

Reply:
yes it is intended, the module is a trick to provide non-namespaced imported target in the old `.names` logic, but it's useless for the new `cmake_target_name` logic.

Reply:
so the intent of this PR is to remove some no-op code ?

Reply:
I think the intent is to remove in CCI any logic based on the previous logic of `set_property()`.